db/re/item_combos.yml

```diff
@@ -25168,4 +25168,470 @@ Body:
+          - aegis_300314     # 300314
+          - Harve_J     # 32029
+      - Combo:
+          - aegis_300314     # 300314
+          - Farthezan_J     # 32354
+      - Combo:
+          - aegis_300314     # 300314
+          - Fortrage_J     # 32030
+      - Combo:
+          - aegis_300314     # 300314
+          - Platinum_Dagger_J     # 28780
+      - Combo:
+          - aegis_300314     # 300314
+          - Black_Circle_J     # 32112
+      - Combo:
+          - aegis_300314     # 300314
+          - Ponitendtia_J     # 26171
+      - Combo:
+          - aegis_300314     # 300314
+          - Boltigin_J     # 28637
+      - Combo:
+          - aegis_300314     # 300314
+          - Combo_Fist_J     # 1868
+      - Combo:
+          - aegis_300314     # 300314
+          - Lucis_Flail_J     # 32407
+    Script: |
+      bonus bBaseAtk,100;
+      bonus bMatk,100;
+      bonus2 bSubRace,RC_Player_Human,10;
+      bonus2 bSubRace,RC_Player_Doram,10;
+      skill "LK_CONCENTRATION",5;
+  - Combos:
+      - Combo:
+          - aegis_300314     # 300314
+          - Judgement_Slasher_J     # 28777
+      - Combo:
+          - aegis_300314     # 300314
+          - Repent_Slasher_J     # 28778
+    Script: |
+      bonus bBaseAtk,200;
+      bonus bMatk,200;
+      bonus2 bSubRace,RC_Player_Human,40;
+      bonus2 bSubRace,RC_Player_Doram,40;
+      skill "LK_CONCENTRATION",5;
+  - Combos:
+      - Combo:
+          - aegis_300314     # 300314
+          - Agudo_Filo_J     # 28049
+      - Combo:
+          - aegis_300314     # 300314
+          - Wind_Gale_J     # 18197
+      - Combo:
+          - aegis_300314     # 300314
+          - Volar_J     # 21056
+      - Combo:
+          - aegis_300314     # 300314
+          - Aiming_Bow_J     # 18195
+      - Combo:
+          - aegis_300314     # 300314
+          - Gravitation_Staff_J     # 2062
+      - Combo:
+          - aegis_300314     # 300314
+          - Sharp_Star_Bow_J     # 18194
+      - Combo:
+          - aegis_300314     # 300314
+          - Adorare_Staff_J     # 2063
+      - Combo:
+          - aegis_300314     # 300314
+          - Staff_Of_Miracle_J     # 2061
+      - Combo:
+          - aegis_300314     # 300314
+          - Falken_Shooter_J     # 18196
+      - Combo:
+          - aegis_300314     # 300314
+          - Vernan_J     # 21057
+      - Combo:
+          - aegis_300314     # 300314
+          - Maxi_Spanner_J     # 28142
+      - Combo:
+          - aegis_300314     # 300314
+          - Rapid_Fire_C_Bow_J     # 18193
+      - Combo:
+          - aegis_300314     # 300314
+          - Reaper_Cross_J     # 28048
+    Script: |
+      bonus bBaseAtk,100;
+      bonus bMatk,100;
+      bonus2 bSubRace,RC_Player_Human,40;
+      bonus2 bSubRace,RC_Player_Doram,40;
+      skill "LK_CONCENTRATION",5;
+  - Combos:
+      - Combo:
+          - Arabian_Manteau     # 480023
+          - Ifrit_Card     # 4430
+    Script: |
+      bonus2 bSubEle,Ele_Fire,100;
```

Comment:
The combo can stay like that but the description of some servers also reduce critical slash by 100% (30% on JRO)

Reply:
Sealed Ifrit Card ( JRO Ver )
I will add card next PR